src/Test/HttpCaller.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Test;
+
+use FOS\HttpCache\Test\HttpClient;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Provides a method for getting responses from your application
+ */
+trait HttpCaller
+{
+    /**
+     * HTTP adapter for requests to the application
+     *
+     * @var HttpClient
+     */
+    protected $client;
```

Comment:
can this be private? what does it mean exactly when a trait has a private property, is it hidden from the class that uses the trait?

from @stof in https://github.com/FriendsOfSymfony/FOSHttpCache/pull/224/files#r33294271